### PR TITLE
feat: adds venus 4626 wrapper

### PR DESCRIPTION
**Pull request by 0xged**

### CODE REVIEW

File: deployments/bnb.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "VenusERC4626Factory": ""
```
Comment (0xsambugs) [inline deployments/bnb.json]: Is this ok? Shouldn't the address be here?

**Review by 0xsambugs — COMMENTED** [review]
May I suggest adding a test like this one?
https://github.com/Mean-Finance/yield-daddy/blob/main/src/test/base/ERC4626Proxy.st.sol

It should test that the standard is implemented correctly and we might find some overflow/underflow limitations

File: deployments/bnb.json (hunk 1 above)
Comment (0xged) [reply deployments/bnb.json]: Sorry! Added file before deployment 😬

**Comment by 0xged** [comment]
> May I suggest adding a test like this one? https://github.com/Mean-Finance/yield-daddy/blob/main/src/test/base/ERC4626Proxy.st.sol
> 
> It should test that the standard is implemented correctly and we might find some overflow/underflow limitations

I think that would be great, but this is an implementation we are already using in prod with Sonne and we've had no issue. I think the trade off of implementing the Comptroller + VERC20 mocks is not worth it. Although... would love to once we have more time.

Do let me know if you feel is a blocker for approval / deployment.

**Review by 0xsambugs — APPROVED** [review]
Great work!

**Pull request merged**